client/test/e2e/services/services.spec.js

```diff
@@ -18,7 +18,7 @@ describe('Services Module', function () {
   var DEFAULT_SERVICE = 4;
   var SERVICE_RANK = helpers.random(DEFAULT_SERVICE);
 
-  var DELETE_SUCCESS = 4;
+  var DELETE_SUCCESS = 5;
```

Comment:
@lomamech My E2E test fails with this new value of 5 but it passes with 4. This test should be updated to ensure it passes.